src/components/Reports/Subscribers/Subscribers.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import { useParams } from 'react-router-dom';
+import { Loading } from '../../Loading/Loading';
+import SafeRedirect from '../../SafeRedirect';
+import SubscriberGdprPermissions from '../GDPRPermissions/SubscriberGdprPermissions';
+import SubscriberSentCampaigns from '../SubscriberSentCampaigns/SubscriberSentCampaigns';
+import { useIntl } from 'react-intl';
+import { Helmet } from 'react-helmet';
+import SubscriberInfo from '../../shared/SubscriberInfo/SubscriberInfo';
+
+const sections = {
+  history: {
+    Component: SubscriberSentCampaigns,
+    titleKey: 'subscriber_history.title',
+    descriptionKey: 'subscriber_history.description',
+  },
+  gdpr: {
+    Component: SubscriberGdprPermissions,
+    titleKey: 'subscriber_gdpr.header_title',
+    descriptionKey: 'subscriber_gdpr.header_description',
+  },
+};
+
+const Subscribers = ({ dependencies: { dopplerApiClient } }) => {
+  const { email, section } = useParams();
```

Comment:
Could you try this with an alias? like `amoschini+test@makingsense.com`?

You should try it reading the email and also generating new links.

Reply:
![image](https://user-images.githubusercontent.com/10213170/86472782-1952ac00-bd16-11ea-8209-357154f63859.png)

Reply:
Great! I cannot understand how it works by default, but, great.

You should have this test into account for the redirection too.